### ISSUE TITLE
Possible Local Order Parameter

### ISSUE THREAD

**Issue by JerryChen97** [JerryChen97/KitaevLadder#33]
1. A table for all the possible ops
2. The overlap between wavefunctions in some phase (remember to avoid the cat states)

**Comment by JerryChen97** [comment]
I designed a function to generate all the possible candidates for (local) order parameters according to the following rules:
 - Commuting with the local symmetries `Dn`
 - Anti-commuting with at least one of the global symmetries `Sigma_x` or `Sigma_y`

And then with its help, the correlation functions of candidates within a given range can be calculated.

**Comment by JerryChen97** [comment]
![IXIY](https://user-images.githubusercontent.com/21050723/93124832-40274a00-f698-11ea-830e-3c613bb71cd0.png)
![IXZX](https://user-images.githubusercontent.com/21050723/93124836-40bfe080-f698-11ea-96b9-6de68a58efa8.png)
![XY](https://user-images.githubusercontent.com/21050723/93124837-40bfe080-f698-11ea-83ae-27b0731e4800.png)
![XYZZ](https://user-images.githubusercontent.com/21050723/93124840-40bfe080-f698-11ea-9745-7d1046fabe99.png)
![XZX](https://user-images.githubusercontent.com/21050723/93124842-40bfe080-f698-11ea-8f17-930167c3d520.png)
![XZYZ](https://user-images.githubusercontent.com/21050723/93124843-40bfe080-f698-11ea-8030-476f7b9c3133.png)
![YIX](https://user-images.githubusercontent.com/21050723/93124845-41587700-f698-11ea-828c-a1d9f0b6cea8.png)
![YIYZ](https://user-images.githubusercontent.com/21050723/93124846-41587700-f698-11ea-827d-f3d1da87fa75.png)
![YX](https://user-images.githubusercontent.com/21050723/93124847-41587700-f698-11ea-8089-99813cc2fcd3.png)
![YXZZ](https://user-images.githubusercontent.com/21050723/93124848-41587700-f698-11ea-8b62-765d712766b2.png)
![ZYIY](https://user-images.githubusercontent.com/21050723/93124849-41587700-f698-11ea-8b66-9397d8ee477d.png)
![ZYZX](https://user-images.githubusercontent.com/21050723/93124851-41f10d80-f698-11ea-8dc9-eb225f9d9e7d.png)
![ZZXY](https://user-images.githubusercontent.com/21050723/93124852-41f10d80-f698-11ea-9536-4a6b5118a348.png)
![ZZYX](https://user-images.githubusercontent.com/21050723/93124853-41f10d80-f698-11ea-947a-6486defc8651.png)

**Comment by JerryChen97** [comment]
The capital letters in the fig above represent local operators: `I := Id, X:= Sigma_x, Y := Sigma_y, Z := Sigma_z`.

**Comment by JerryChen97** [comment]
Though quite inefficient, it can also generate any arbitrary range (w.r.t. how many unicells on the chain the local operator involves).
I also tested all range-3 operators and all vanished.

**Comment by JerryChen97** [comment]
P.S. The MPS I used was generated by periodic DMRG with 80 physical spin-1/2 sites.

**Comment by JerryChen97** [comment]
@aaronszasz from these results I just don't believe the existence of any local order parameter.

However, I am still unable to exclude this possibility in theory. I am wondering if anyone else has tried showing that in a 1D local spin system, the range of any possible local order parameter operator will have an upper bound scaling with the range of the system's interaction (e.g. I will say the range in our Kitaev ladder to be 3); if this is true, then we don't have to show all the vanishing results till infinity, which is obviously impossible.

**Comment by JerryChen97** [comment]
Right now I am thinking about the other possible explanation: such a Kitaev ladder may be an SPT.
But I have no clue how to prove this either numerically or theoretically.

**Comment by aaronszasz** [comment]
I'm not aware of such a result, but that doesn't mean there isn't one.  I haven't really put any time into thinking about this question, but I can do so this week to hopefully have something useful to say.

**Comment by aaronszasz** [comment]
-> By "this question" I mean the existence of a local order parameter in this model

**Comment by JerryChen97** [comment]
> I'm not aware of such a result, but that doesn't mean there isn't one. I haven't really put any time into thinking about this question, but I can do so this week to hopefully have something useful to say.

Yeah definitely I didn't show that there is absolutely no local order parameter in this model. Right now I can only say that this is my own guess; btw the property of being an SPT and that of being an SSB should not be contradicting right?

By running over all the possible range-5 candidates (involving up-to-10 spin-1/2 sites) satisfying the two rules above, I found the total number is 7168; this is not a huge number in principle but it takes some time to check all of them.
In a word, it is almost impossible to confidently claim that there is no such stuff if it does not exist.

**Comment by JerryChen97** [comment]
Here is a map of the XY local order parameter generated from the ~20k points I've got recently.

![image](https://user-images.githubusercontent.com/21050723/95125988-c1b44a00-0723-11eb-9bd0-0f984b89369b.png)

**Comment by JerryChen97** [comment]
![image](https://user-images.githubusercontent.com/21050723/95126054-d98bce00-0723-11eb-9cd8-65efd477260d.png)

**Comment by JerryChen97** [comment]
![image](https://user-images.githubusercontent.com/21050723/95126095-e4def980-0723-11eb-81a9-0649a7c197ae.png)

**Comment by JerryChen97** [comment]
@aaronszasz I hope you enjoy the 3 figures above!

**Comment by aaronszasz** [comment]
Looks great!

**Comment by aaronszasz** [comment]
This is indeed exactly what I was asking to see, thanks!

**Comment by aaronszasz** [comment]
I'm actually a bit surprised that it's so large even at Jx=Jy=2.  Very neat

**Comment by JerryChen97** [comment]
> This is indeed exactly what I was asking to see, thanks!

Your idea is excellent and also matches perfectly with my iDMRG simulation! I am also working on the large-X limit derivation; will update it on Overleaf and email you and Yin-Chen once it's done~

**Comment by aaronszasz** [comment]
Great, I look forward to seeing it!

**Comment by aaronszasz** [comment]
How is the large-X perturbation theory coming?

**Comment by JerryChen97** [comment]
I think I've got the reason why the OBC system is two-fold degenerate instead of 4: the cluster model our Kitaev ladder corresponds to is slightly different from the model mentioned in Xie's textbook.
A ring of the cluster model with `N` spins is composed of `N` local cluster Hamiltonian `ZXZ`, but when the ring is broken into an open chain, Xie's treatment is getting rid of two of those `ZXZ`, corresponding to the start and the end of the chain; therefore, in their OBC cluster model, there will be a four-fold degeneracy.
However, in our case, the cluster model is induced from the original ladder, where only one of the effective `ZXZ` is lost when the PBC is changed to OBC; so we will only get a two-fold degeneracy. I am still creating some figures to illustrate this point, and I will attach them to the Overleaf doc and email you and Yin-Chen.

Still, I couldn't well explain another problem: why the PBC Kitaev ladder still seems to be 2-fold degenerate in the large-X limit? According to the effective Hamiltonian, there should be no more degeneracy. I am still checking if something in my code went wrong.

**Comment by aaronszasz** [comment]
Did you try Yin-Chen's suggestion to do the PBC case in finite-system DMRG and see if you still get the degeneracy?

**Comment by JerryChen97** [comment]
> Did you try Yin-Chen's suggestion to do the PBC case in finite-system DMRG and see if you still get the degeneracy?

Yes, I tried and there is no more degeneracy. Also, I did the matching for smaller sizes (8, 12 and 16 spins) between fDMRG of PBC with Exact Diagonalization on the energy and it seems perfect.

**Comment by aaronszasz** [comment]
So it sounds like your first excited state energy from the analytical method is wrong?

**Comment by JerryChen97** [comment]
I also did the benchmark between the numerical integral over the spectra and ED, and an interesting fact is: for GS energy, the numerical integral works perfectly for any finite PBC, but not always good for the first excited energy; it doesn't work when the system is around the large-X limit.

**Comment by JerryChen97** [comment]
> So it sounds like your first excited state energy from the analytical method is wrong?

Exactly. I am still wondering about the essential reason, but I couldn't get a very good answer to this. I will share and discuss with you and Yin-Chen hours later.

**Comment by aaronszasz** [comment]
Ok, I'll look forward to hearing about it later then.
